Classes/Issues/Merge/IssueMergeSectionController.swift

```diff
@@ -65,13 +65,25 @@ MergeButtonDelegate {
         var viewModels = [ListDiffable]()
 
         if object.contexts.count > 0 {
-            let success = object.contexts.reduce(true, { $0 && $1.state == .success })
-            viewModels.append(IssueMergeSummaryModel(
-                title: success ?
-                    NSLocalizedString("All checks passed", comment: "")
-                    : NSLocalizedString("Some checks failed", comment: ""),
-                state: success ? .success : .failure
-            ))
+            let states = object.contexts.map { $0.state }
+            let state: IssueMergeSummaryModel.State
+            let stateDescription: String
+            let containsFailure = states.contains(.failure) || states.contains(.error)
+            switch states {
+            case _ where containsFailure:
+                state = .failure
+                stateDescription = NSLocalizedString("Some checks failed", comment: "")
+            case let states where states.contains(.pending) && !containsFailure:
+                state = .pending
+                stateDescription = NSLocalizedString("Merge status pending", comment: "")
```

Comment:
GitHub uses `[waiting to hear about {commit}](https://user-images.githubusercontent.com/4190298/36353428-a37fb9cc-14c6-11e8-9a04-91fdc9f95f10.jpeg)`, but couldn't find any last-commit info so changed it to this.

Reply:
Fine w/ this. Less text is better since we have less screen to work w/, and the `UILabel` is limited to one line w/ truncation.